sdk/monitor/Azure.Monitor.Ingestion/src/Azure.Monitor.Ingestion.csproj

```diff
@@ -4,7 +4,7 @@
     <AssemblyTitle>Azure Monitor Ingestion client library</AssemblyTitle>
     <Version>1.0.0-beta.5</Version>
     <PackageTags>Azure Monitor Ingestion</PackageTags>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net461;net5.0</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

Comment:
CI was no longer passing due to `net5.0` being out of support.   These look like they were added as part of a POC for GZip support and should no longer be neded.

_([ref](https://github.com/Azure/azure-sdk-for-net/pull/29426))_